service/src/main/resources/api/openapi.yml

```diff
@@ -496,6 +522,62 @@ components:
         failures:
           $ref: '#/components/schemas/FailureMessage'
 
+    SubmitWorkflowRequestBody:
+      type: object
+      properties:
+        workflowGcsUri:
+          description: GCS uri which points to the workflow. Either workflow GCS uri or workflow url is required. If both are specified, then workflow URL takes precedence.
+          type: string
+        workflowUrl:
+          description: URL which points to the workflow. Either workflow GCS uri or workflow url is required. If both are specified, then workflow URL takes precedence.
+          type: string
+        workflowOnHold:
+          description: Put workflow on hold upon submission. By default, it is taken as false.
+          type: boolean
+          default: false
+        workflowInputs:
+          description: JSON or YAML file containing the inputs as an object. For WDL workflows a skeleton file can be generated from WOMtool using the "inputs" subcommand.
+          type: string
+        workflowOptions:
```

Comment:
I've included an outline of some options we could possibly specify; however, currently this can accept arbitrary options too (`additionalProperties: true`).